tests/lib/validator_test.py

```diff
@@ -290,6 +291,24 @@ def _create_route1(self):
         )
 
 
+class TestVxlanValidation(object):
+    def test_invalid_vxlan_without_parent(self):
```

Comment:
Reads better I think if you call this `test_missing_parent_is_invalid`.
The use of `vxlan` in the test name is redundant, you have the class name to mention that.